lib/ar_state_machine/version.rb

```diff
@@ -1,3 +1,3 @@
 module ARStateMachine
-  VERSION = "0.1.6"
+  VERSION = "0.1.8"
```

Comment:
needs bundle install

Reply:
i think i did that right before you said dis